### ISSUE TITLE
mariadb version string where "mariadb" is embedded within additional characters

### ISSUE THREAD

**Issue by teli5200** [sqlalchemy/sqlalchemy#4624]
My version number is 10.1.9-MariaDBV1.0R050D002-20170809-1522, the following code is recognized as mysql, So I will report  error 

`sqlalchemy.exc.OperationalError: (MySQLdb._exceptions.OperationalError) (1193, "Unknown system variable 'transaction_isolation'")`

thank you

**Comment by zzzeek** [comment]
hi - 

what's the source of this version string, is this just if you build form source ?

**Comment by sqla-tester** [comment]
**Mike Bayer** has proposed a fix for this issue in the **master** branch:

**Add additional parsing to extract the "MariaDB" keyword from versinon string** https://gerrit.sqlalchemy.org/1201

**Comment by zzzeek** [comment]
hi -

why are you closing this?  I wrote a fix that's going through testing.

**Comment by yangkang55** [comment]
i have met the same MariaDBV1 problem ,why not fix it

**Comment by zzzeek** [comment]
it is fixed but I forgot to backport it

**Comment by sqla-tester** [comment]
**Mike Bayer** has proposed a fix for this issue in the **rel_1_3** branch:

**Add additional parsing to extract the "MariaDB" keyword from version string** https://gerrit.sqlalchemy.org/1343

**Comment by zzzeek** [comment]
this will be in the next release sorry I missed it
